### PR TITLE
Model generation

### PR DESCRIPTION
**Pull request by MauricioUyaguari**

### CODE REVIEW

**Review by kevin-m-knight-gs — COMMENTED** [review]
It looks like you moved TestFileGenerationFactory.java into a directory named "org.finos.legend.sdlc.generation.file" instead of "org/finos/legend/sdlc/generation/file".

**Comment by MauricioUyaguari** [comment]
> It looks like you moved TestFileGenerationFactory.java into a directory named "org.finos.legend.sdlc.generation.file" instead of "org/finos/legend/sdlc/generation/file".

Fixed. thanks

File: legend-sdlc-generation-model/src/test/java/org/finos/legend/sdlc/generation/model/TestModelGenerationFactory.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        factory.processModelGenerator(simpleClassGenerator);
+        try
+        {
+            factory.validateAndBuildGeneratedModel();
+            Assert.fail("Expected a validation error but no error occurred");
+        }
+        catch (Exception error)
+        {
+            Assert.assertEquals("Generated element 'model::MyClass' of type Class can't override existing element of type Class", error.getMessage());
+        }
```
Comment (kevin-m-knight-gs) [inline legend-sdlc-generation-model/src/test/java/org/finos/legend/sdlc/generation/model/TestModelGenerationFactory.java:163]: Can we use Assert.assertThrows for this?

Comment (kevin-m-knight-gs) [reply legend-sdlc-generation-model/src/test/java/org/finos/legend/sdlc/generation/model/TestModelGenerationFactory.java:163]: And, actually, that comment should apply to more than just that one place. It should apply to all the places with that same pattern.

**Review by kevin-m-knight-gs — APPROVED** [review]

**Pull request merged**